RealmTasks Android/app/src/main/java/io/realm/realmtasks/TaskActivity.java

```diff
@@ -155,20 +155,17 @@ public void onMoved(RecyclerView recyclerView, ItemViewHolder from, ItemViewHold
             final int fromPosition = from.getAdapterPosition();
             final int toPosition = to.getAdapterPosition();
             adapter.onItemMoved(fromPosition, toPosition);
-            adapter.notifyItemMoved(fromPosition, toPosition);
         }
 
         @Override
         public void onCompleted(ItemViewHolder viewHolder) {
             adapter.onItemCompleted(viewHolder.getAdapterPosition());
-            adapter.notifyDataSetChanged();
         }
 
         @Override
         public void onDismissed(ItemViewHolder viewHolder) {
             final int position = viewHolder.getAdapterPosition();
             adapter.onItemDismissed(position);
-            adapter.notifyItemRemoved(position);
```

Comment:
When one row is removed, other items have different colors. So it should be called here.